components/authentication/base/segment-bridge.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: segment-bridge-read-access
   namespace: toolchain-host-operator
 rules:
-- apiGroups: [""]
+- apiGroups: ["toolchain.dev.openshift.com"]
   resources: ["UserSignup"]
```

Comment:
resources must be in plural and lower case -> usersignups (as far as I know)

Reply:
You're right, fixed.